echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/InteractiveNotificationCallbackHandler.groovy

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Do not add new Groovy files to the main source set.

Reply:
OK, OK...👮 It was a single file and there were not any existing Java sources in this module, so I thought it was not a big deal. But will convert.